.github/workflows/go-tests.yml

```diff
@@ -24,8 +24,23 @@ env:
   GOPRIVATE: github.com/hashicorp # Required for enterprise deps
 
 jobs:
+  conditional-skip:
+    runs-on: ubuntu-latest 
+    name: Get files changed and condtionally skip CI
+    outputs:
+      files-changed: ${{ steps.read-files.outputs.trigger-ci }}
+    steps:
+      - uses: actions/checkout@v3
```

Comment:
change to https://github.com/hashicorp/consul/blob/5b88aae3b4119ccf20674d57907254cecdfaca65/.github/workflows/go-tests.yml#L36 to make it consistent